lightning/restore/tidb.go

```diff
@@ -302,14 +304,11 @@ func AlterAutoIncrement(ctx context.Context, db *sql.DB, tableName string, incr
 	return errors.Annotatef(err, "%s", query)
 }
 
-func AlterAutoRandom(ctx context.Context, db *sql.DB, tableName string, randomBase int64) error {
-	sql := common.SQLWithRetry{
-		DB:     db,
-		Logger: log.With(zap.String("table", tableName), zap.Int64("auto_random", randomBase)),
-	}
+func AlterAutoRandom(ctx context.Context, g glue.SQLExecutor, tableName string, randomBase int64) error {
+	logger := log.With(zap.String("table", tableName), zap.Int64("auto_random", randomBase))
```

Comment:
will using `log.With` use TiDB's own logger?

Reply:
currently not. in following PR will add a `RunEmbeddedOnce(context.Context, config.Config, zap.Logger)` that replace `appLogger` with given logger